modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.tsx

```diff
@@ -310,6 +339,7 @@ interface IProps {
 	inputMaskFormat?: string;
 	localizedValue?: LocalizedValue<string>;
 	name: string;
+	numericInputMask: string;
```

Comment:
Since you are checking in [#L114](https://github.com/liferay-forms/liferay-portal/blob/370cc8cf3b4a8a3f5953053d482e0e2870a96e59/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.tsx#L114) if the `numericInputMask` is a `string`, it probably can be something else.  Update the type here or remove the `if` clause there if it can't be anything else but a `string`.